content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -234,7 +235,8 @@ Additionally, the following restrictions are applied to the schema:
 - The field `uniqueItems` cannot be set to true.
 - The field `additionalProperties` cannot be set to false.
 
-This feature is __beta__ in v1.9.
+{{< feature-state state="beta" for_kubernetes_version="1.9" >}}
```

Comment:
I think this would be better added just after  the`Validation` title, and then go on with the details.

Reply:
This also need to be edited for subresources.

Reply:
@mistyhacks leaving this one to you.